.github/workflows/roxy.yml

```diff
@@ -0,0 +1,66 @@
+name: "Deploy MarkLogic config"
```

Comment:
The name here and the name of this file don't feel like they match? Which maybe they do, I just don't really know what roxy is so if I was looking for the github action that deploys marklogic I wouldn't guess the file would be called `roxy.xml`

Reply:
I'll update it - for reference Roxy is the tool that updates MarkLogic with the version controlled config + code i.e. its "./ml"